src/main/java/hng_java_boilerplate/config/WebSecurityConfig.java

```diff
@@ -71,7 +72,7 @@ public SecurityFilterChain httpSecurity(HttpSecurity httpSecurity) throws Except
                 .csrf(AbstractHttpConfigurer::disable)
                 .authorizeHttpRequests(httpRequests ->
                         httpRequests
-//                                .requestMatchers("").hasAuthority(String.valueOf(Role.ADMIN))
+                                .requestMatchers("/api/v1/blog/edit/**").hasAuthority(String.valueOf(Role.ROLE_SUPERADMIN))
```

Comment:
You can merge this with the ADMIN by using `hasAnyAuthority`. Also, I don't think you need to specify the endpoint here.

Reply:
Alright sir